DISCLAIMER

```diff
@@ -1,6 +1 @@
-Apache Hudi (incubating) is an effort undergoing incubation at The Apache Software Foundation
-(ASF), sponsored by the Apache Incubator PMC. Incubation is required of all newly accepted
-projects until a further review indicates that the infrastructure, communications, and decision
-making process have stabilized in a manner consistent with other successful ASF projects. While
-incubation status is not necessarily a reflection of the completeness or stability of the code,
-it does indicate that the project has yet to be fully endorsed by the ASF.
+Apache Hudi (incubating) is an effort undergoing incubation at The Apache Software Foundation (ASF), sponsored by the Apache Incubator PMC. Incubation is required of all newly accepted projects until a further review indicates that the infrastructure, communications, and decision making process have stabilized in a manner consistent with other successful ASF projects. While incubation status is not necessarily a reflection of the completeness or stability of the code, it does indicate that the project has yet to be fully endorsed by the ASF.
```

Comment:
sorry for back and forth. but seems like review explicitly asked for -WIP disclaimer? Safer to do that itself?